config/prow/cluster/monitoring/base-prow/grafana-deployment.yaml

```diff
@@ -8,6 +8,7 @@ spec:
     spec:
       containers:
       - name: grafana
+        image: grafana/grafana:7.5.15
```

Comment:
If you want, you can use the `images` field in kustomize to overwrite the image ref in a central place :)
https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/images/

Reply:
it works 😄